query-execution-plan.md

```diff
@@ -8,156 +8,471 @@ aliases: ['/docs/dev/query-execution-plan/','/docs/dev/reference/performance/und
 
 Based on the details of your tables, the TiDB optimizer chooses the most efficient query execution plan, which consists of a series of operators. This document details the execution plan information returned by the `EXPLAIN` statement in TiDB.
 
-## Optimize SQL statements using `EXPLAIN`
+## `EXPLAIN` overview
 
 The result of the `EXPLAIN` statement provides information about how TiDB executes SQL queries:
 
-- `EXPLAIN` works together with `SELECT`, `DELETE`, `INSERT`, `REPLACE`, and `UPDATE`.
-- When you run the `EXPLAIN` statement, TiDB returns the final physical execution plan which is optimized by the SQL statement of `EXPLAIN`. In other words, `EXPLAIN` displays the complete information about how TiDB executes the SQL statement, such as in which order, how tables are joined, and what the expression tree looks like. For more information, see [`EXPLAIN` output format](#explain-output-format).
-- TiDB also supports `EXPLAIN [options] FOR CONNECTION connection_id`, but with minor differences from MySQL. See [`EXPLAIN FOR CONNECTION`](#explain-for-connection).
+- `EXPLAIN` works together with statements such as `SELECT` and `DELETE`.
+- When you run the `EXPLAIN` statement, TiDB returns the final physical execution plan which is optimized by the SQL statement of `EXPLAIN`. In other words, `EXPLAIN` displays the complete information about how TiDB executes the SQL statement, such as in which order, how tables are joined, and what the expression tree looks like.
```

Comment:
which is optimized by the SQL statement of `EXPLAIN` -> which is optimized by the optimizer